public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -163,7 +166,15 @@ export const AlertRuleForm: FC<Props> = ({ existing, prefill }) => {
         type: RuleFormType.grafana,
       });
     }
-  }, [defaultDsAndQueries.queries, reset, existing, prefill, defaultsInQueryParamsObject]);
+  }, [
+    defaultDsAndQueries.queries,
+    reset,
+    existing,
+    prefill,
+    defaultsInQueryParamsObject,
+    returnTo,
```

Comment:
A bit confused why `returnTo` needs to be in the dependencies array for this `useEffect`

Reply:
ohh no, you're right, I forgot to remove it